### ISSUE TITLE
Failed to launch gale

### ISSUE THREAD

**Issue by UnsaidRaine** [Kesomannen/gale#180]
says its trying to find data for Shadows of doubt, which isnt a game I own, says it cant find its json manifest? all I did was update the launcher

**Comment by Kesomannen** [comment]
Weird. You should be able to fix it by removing the shadows-of-doubt folder inside of gale's data folder.

**Comment by UnsaidRaine** [comment]
Yeah I just ran the installer to remove gale and reinstall and its working again
